### PR TITLE
Simple recommender

### LINKED ISSUE #4: Implement recommender service GUI.
A simple gui that allows to see recommendation for a given twitter/wykop.pl user.

### PR DESCRIPTION
**Pull request by Mequrel**
Implemented a simple GUI for recommendation purposes. The model building will be added in the next  PR. That closes #4.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -46,6 +46,25 @@ libraryDependencies += "com.datastax.cassandra" % "cassandra-driver-core" % "2.1
 
 libraryDependencies += "com.fasterxml.jackson.core" % "jackson-core" % "2.5.2"
 
+libraryDependencies ++= {
+  val sprayVersion = "1.3.3"
```
Comment (Adebski) [inline build.sbt]: I would move this value (and other library versions if neded be) to somethng like project/Versions.scala, then create an object Verisons and put version number there

File: src/main/scala/ztis/recommender/Recommender.scala [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+object Recommender extends App with StrictLogging {
+  val config = ConfigFactory.load("recommender")
+
+  implicit val system = ActorSystem("on-spray-can", config)
+
+  val service = system.actorOf(Props[RecommenderPortActor], "demo-service")
+
+  implicit val timeout = Timeout(5.seconds)
+  IO(Http) ? Http.Bind(service, interface = "localhost", port = 8080)
```
Comment (Adebski) [inline src/main/scala/ztis/recommender/Recommender.scala]: I would extract port to configuration so it is possible to change it when starting the application, the same with the interface (or use "0.0.0.0")

**Comment by Adebski** [comment]
LGTM

File: build.sbt (hunk 1 above)
Comment (Mequrel) [inline build.sbt]: Done

File: src/main/scala/ztis/recommender/Recommender.scala (hunk 2 above)
Comment (Mequrel) [inline src/main/scala/ztis/recommender/Recommender.scala]: Done

**Comment by Mequrel** [comment]
Closes #4

**Pull request merged**